src/utils/api/getAccount.ts

```diff
@@ -7,9 +7,11 @@ import { BASEURL } from "./constants";
  *   account=NXT-4VNQ-RWZC-4WWQ-GVM8S
  */
 
-function getAccount(account: string) {
+async function getAccount(account: string) {
+  let result;
   try {
-    API(`${BASEURL}requestType=getAccount&account=${account}`, "GET");
+    result = await API(`${BASEURL}requestType=getAccount&account=${account}`, "GET");
```

Comment:
the call to the API should not require the BASEURL to be provided. it should prepend it.

or better yet, have an NXT_API call that you import which is only responsible for prepending the jup api and then this pattern will allow for future additional external api's